ming_sim/action_materialize.py

```
@@ -75,6 +75,35 @@ def _draft_path_took_effect(ctx: MaterializeCtx) -> bool:
     return False
 
 
+def _materializable_draft_xiexang(
+    ctx: MaterializeCtx,
+    candidate: Dict[str, Any],
+) -> Dict[str, Any]:
+    """在真实写入前置条件齐全时，把 draft 协饷投影为本轮 grant 候选。
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)